Cargo.toml

```diff
@@ -14,4 +14,4 @@ categories = ["no-std", "rust-patterns", "parsing"]
 byteorder = "1"
 
 [dev-dependencies]
-struct_deser-derive = "0.1.0"
+struct_deser-derive = {path = "struct_deser-derive"}
```

Comment:
You forgot to revert this.

Reply:
But this is dev-dependencies . What is disadvantage of this over use version from crates.io?

Reply:
In the next PR I would like re-export derive: https://github.com/xoac/struct_deser/blob/edition/src/lib.rs#L166

Reply:
Ah, sorry, I missed that. Having dev-dependency as path dependency sounds useful. You may keep it.